### DISCUSSION TITLE
Nuxt theme router case sensitive

### DISCUSSION THREAD

**Discussion by diacu** [DivanteLtd/vue-storefront#6629] [Nuxt-Theme]
https://github.com/vuestorefront/vue-storefront/blob/5765b5093a4f0add70b271e75820580f7aa5bafc/packages/core/nuxt-theme-module/index.js#L26

I know that it doesn't have much of an impact, but maybe we should remove the standard Nuxt generated routes when generating new ones form nuxt-theme? Because I see duplicate routes in the debugger. See my screen bellow:
<img width="458" alt="Screenshot 2022-02-08 at 16 39 20" src="https://user-images.githubusercontent.com/1351863/153010055-19afdd39-a0bc-4b38-8429-5c115212fb68.png">

**Comment by filrak** [comment]
Yeah, this is definitely something worth doing in all integrations, thanks for reporting! Can you create an issue so we can add it to the backlog?

**Comment by filipsobol** [comment]
I absolutely agree and believe we should use [File-based routing](https://nuxtjs.org/docs/features/file-system-routing/) whenever possible
